myst_nbs/case_studies/stochastic_volatility.myst.md

```diff
@@ -182,9 +189,25 @@ axes[1].set_title("Posterior volatility");
 
 ## References
 
-1. Hoffman & Gelman. (2011). [The No-U-Turn Sampler: Adaptively Setting Path Lengths in Hamiltonian Monte Carlo](http://arxiv.org/abs/1111.4246).
+:::{bibliography}
+:filter: docname in docnames
+:::
 
 ```{code-cell} ipython3
 %load_ext watermark
 %watermark -n -u -v -iv -w -p aesara,xarray
 ```
+
+## Authors
+
++++
+
+* Written by Thomas Wiecki in 2013
```

Comment:
Notebook seems to have been created in https://github.com/pymc-devs/pymc/commit/f76a6b151dcda7a77193125c224030cdccef7032 from the already existing pyhton script with that example. The python script looks like it was introduced in https://github.com/pymc-devs/pymc/commit/2fe6e88ebb31c201bd18c0cefa1b9840d4fb3911

Reply:
Thanks for tracking down the history! Did you find it just by going to an older version of `pymc` and then looking at the history from there? Or is there a trick to viewing the file history back to 2013 on GitHub from within the `pymc-examples`?

Reply:
Went to pymc repo and searched "stochastic volatility" on the PR tab, opened the oldest one and clicked on "browse files" or something like that which brought me to the right notebook already around 2013, then checked the history of that file to find the first commit I linked. From there I checked the history of the python script to find the 2nd commit I linked. I hope that makes sense, seeing it written sounds super complicated but it took longer to write this down than it took to do the actual thing :sweat_smile: